src/main/java/frc/robot/subsystems/SwagLights.java

```diff
@@ -4,38 +4,52 @@
 import edu.wpi.first.wpilibj2.command.Subsystem;
 
 public class SwagLights implements Subsystem {
+
+    private Acquisition acquisition = Acquisition.getInstance();
+    private Storage storage = Storage.getInstance();
+
+    private boolean noteSeenAcquire = false;
+    private boolean noteSeenStorage = false;
+
     // Enums
     public enum RobotStates {
         Enabled("A"),
         Disabled("D"),
         EmergencyStop("E");
 
-        public final String value;
```

Comment:
Why change this to private? 
since it’s final I see no reason to add the extra code to make a getter function